### PR TITLE
Fix #314

### LINKED ISSUE #314: UnicodeDecodeError
Dear AIL team!

./Phone.py
./Bitcoin.py:
./Lines.py:
./RegexForTermsFrequency.py
./SentimentAnalysis.py
./Keys.py
./Categ.py
./Indexer.py
./BankAccount.py
./SetForTermsFrequency.py
./Decoder.py
./DomClassifier.py
./Release.py
./ModuleStats.py

Queues stop work with the following error like: "UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb7 in position 8242: invalid start byte"

eg. the Phone.py Traceback:
Traceback (most recent call last):
  File "./Phone.py", line 85, in <module>
    search_phone(message)
  File "./Phone.py", line 24, in search_phone
    content = paste.get_p_content()
  File "/opt/AIL-framework/bin/packages/Paste.py", line 128, in get_p_content
    paste = self.cache.get(self.p_path)
  File "/opt/AIL-framework/AILENV/lib/python3.5/site-packages/redis/client.py",              line 976, in get
    return self.execute_command('GET', name)
  File "/opt/AIL-framework/AILENV/lib/python3.5/site-packages/redis/client.py",              line 668, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/opt/AIL-framework/AILENV/lib/python3.5/site-packages/redis/client.py",              line 680, in parse_response
    response = connection.read_response()
  File "/opt/AIL-framework/AILENV/lib/python3.5/site-packages/redis/connection.p             y", line 624, in read_response
    response = self._parser.read_response()
  File "/opt/AIL-framework/AILENV/lib/python3.5/site-packages/redis/connection.p             y", line 326, in read_response
    response = self.encoder.decode(response)
  File "/opt/AIL-framework/AILENV/lib/python3.5/site-packages/redis/connection.p             y", line 125, in decode
    value = value.decode(self.encoding, self.encoding_errors)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 122130: inv             alid continuation byte

I using redis-py 2.10.6 version.

### PR DESCRIPTION
**Pull request by kovacsbalu**
Replace char on redis encoding error.
Try to use local file on other error.

### CODE REVIEW

**Comment by Terrtia** [comment]
Hey @kovacsbalu 

Thank you for your contribution !

Do you succeed to reproduce this error ?
Can you provide an example of a paste ?

**Comment by kovacsbalu** [comment]
I could reproduce with this script and this paste:
[https://pastebin.com/BbMkEM84](https://pastebin.com/BbMkEM84)
```
#!/usr/bin/env python3
# -*-coding:UTF-8 -*

from packages import Paste

msg = "/opt/AIL-framework/PASTES/archive/pastebin.com_pro/2019/05/14/BbMkEM84.gz"
PST = Paste.Paste(msg)
print(PST.get_p_content())
```

File: bin/packages/Paste.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ def __init__(self, p_path):
             host=cfg.get("Redis_Queues", "host"),
             port=cfg.getint("Redis_Queues", "port"),
             db=cfg.getint("Redis_Queues", "db"),
+            encoding_errors='replace',
```
[the remark this answers is not in the document]
Comment (kovacsbalu) [reply bin/packages/Paste.py]: Ok, removed.

[the remark this answers is not in the document]
Comment (Terrtia) [reply bin/packages/Paste.py]: Great, thanks !

**Comment by Terrtia** [comment]
Thank you !

**Pull request merged**